### ISSUE TITLE
$dotcontent.find() complains about a contentlet having multiple live versions.

### ISSUE THREAD

**Issue by x0rsw1tch** [dotCMS/core#19451]
**Describe the bug**

> Note: Cloud instance (5.3.8)

When scoping a contentlet/widget using the method `#set ($content = $dotcontent.find($contentletId))`, and the contentlet has multiple language versions, dotCMS logs a WARN with the following message:

```
WARN util.ContentUtils: More than one live or working content found with identifier = xxxxxxxx-xxx-xxxx-xxxx-xxxxxxxxxxxx @ url:GET*********************** | lang:1 | ip:***.***.***.*** | Admin:false | start:10-14-2020 04:35:22 EDT
```

This gets incredibly annoying when debugging or troubleshooting an issue with dozens of these messages appearing every few seconds.

**To Reproduce**
1. Create a widget instance of a Widget Type having multiple language versions.
2. In widget code, use the `#set ($content = $dotcontent.find($contentletId))` to scope the content object.
3. See error

**Expected behavior**

dotcms does not complain about a contentlet with multiple live inodes.

**Screenshots**

![image](https://user-images.githubusercontent.com/9543257/96051160-f1accd00-0e48-11eb-9a29-8b2fbcac9989.png)

**Additional context**

Maybe add a separate `find()` method with a language argument?

**Comment by wezell** [comment]
Test:

- TimeMachine
- Multilingual
- `ab` test against a page using this code

**Comment by bryanboza** [comment]
Fixed, tested on release-21.11 // Docker and now it is working as expected. Also new cards reported on #21228 and #21226 

Here the ab testing results:

![image](https://user-images.githubusercontent.com/2641437/140426067-511711bf-8fbf-4392-a251-023429d5440b.png)
